pkg/vfsgen/vfsgen_data.go

```diff
@@ -1583,42 +1583,42 @@ var Assets = func() http.FileSystem {
 		},
 		"/profiles": &vfsgen۰DirInfo{
 			name:    "profiles",
-			modTime: time.Date(2019, 7, 29, 23, 17, 29, 916093878, time.UTC),
+			modTime: time.Date(2019, 7, 30, 22, 26, 48, 807533943, time.UTC),
```

Comment:
Do we need to update the vfs gen data every time?

Reply:
No, only after charts change. This is just incidental but doesn't cause harm.